src/components/CheckEditor/testHelpers.ts

```diff
@@ -18,11 +18,10 @@ export const toggleSection = async (sectionName: string, user: UserEvent): Promi
   return sectionHeader.parentElement?.parentElement ?? new HTMLElement();
 };
 
-export const submitForm = async (onReturn: (arg0: Boolean) => void, user: UserEvent) => {
+export const submitForm = async (user: UserEvent) => {
```

Comment:
I thought what `onReturn` was doing was a little odd (you'll see I've co-located it to the onSuccess methods and using the locationService.goBack method -- as there are multiple places you can edit a check from now) and I didn't like the prop-drilling where it was being continuously passed down.